src/include/daos/mgmt.h

```diff
@@ -76,5 +77,7 @@ ssize_t dc_mgmt_sys_encode(struct dc_mgmt_sys *sys, void *buf, size_t cap);
 ssize_t dc_mgmt_sys_decode(void *buf, size_t len, struct dc_mgmt_sys **sysp);
 int dc_mgmt_get_pool_svc_ranks(struct dc_mgmt_sys *sys, const uuid_t puuid,
 			       d_rank_list_t **svcranksp);
+int dc_mgmt_pool_connect(struct dc_pool *pool);
+int dc_mgmt_pool_disconnect(struct dc_pool *pool);
```

Comment:
These methods should mention "monitor" in the name, to avoid confusion with the actual dc_pool_connect/disconnect methods.

Reply:
Added monitor_ between mgmt_ and pool